src/test/java/StringManipulationTest.java

```diff
@@ -87,27 +130,66 @@ public void testGeSubStrings1() {
         assertEquals(sStings[1], "string");
     }
```

Comment:
you're testing the splitting of a string into substrings. This is a good test case. Have you considered what might happen if the string contains multiple spaces between words?